ansible/host_vars/firefly/cloudflared.yml

```diff
@@ -0,0 +1,10 @@
+---
+additional_ingress:
+  - hostname: "firefly.{{ domain }}"
+    service: "http://localhost:{{ firefly_port }}"
+    originRequest:
+      httpHostHeader: "firefly.{{ domain }}"
+  - hostname: "firefly-importer.{{ domain }}"
+    service: "http://localhost:{{ firefly_fidi_port }}"
+    originRequest:
+      httpHostHeader: "firefly-importer.{{ domain }}"
```

Comment:
This comment (https://github.com/jon77p/infrastructure/pull/104#issuecomment-1100595454) could be fixed here.